apps/cloudbroker/base/cloudapi__users/specs/actors.spec

```diff
@@ -3,7 +3,7 @@
 	User management
 	"""
 
-	method:authenticate @noauth
+    method:authenticate @noauth
```

Comment:
changes in whitespaces? You sure its still loads? Actor spec parser is very picky about these kind of things

Reply:
Yes, it is more consistent with the rest of the methods in the spec as well.